kolibri/plugins/management/assets/src/views/index.vue

```diff
@@ -9,7 +9,7 @@
       <component class="manage-content page" :is="currentPage"/>
     </div>
 
-    <auth-message v-else authorizedRole="admin" />
+    <auth-message v-else :details="$tr('admninOrSuperuser')" />
```

Comment:
Typo here? `adminin` == `admin`. Check to see if it matches `options.$trs` too.

Reply:
> Check to see if it matches `options.$trs` too.

Can you clarify?